src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -352,6 +357,9 @@ class ServiceDetails extends React.Component<ServiceDetailsProps, ServiceDetails
                     </>
                   </NavItem>
                 ))}
+              {this.state.serviceDetailsInfo.apiDocumentation.baseUrl && (
```

Comment:
What happens here if `apiDocumentation` is null or the base url is `''`? And also, why does it get it from the state instead of using the proper props from Redux?

Reply:
If apiDocumentation is not defined the tab is not displayed (it is probably displayed if baseUrl is '' but that shouldn't happen)

Reply:
If we're checking for presence on those maybe a good idea would be just to tag it as nullable, and then check on the whole object.

Reply:
Ok, I will add a check on the full object